client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -27,6 +27,12 @@ import Gravatar from 'components/gravatar';
 import i18n from 'lib/mixins/i18n';
 
 /**
+ * Constants
+ */
+
+const STATS_PAGE = '/stats/insights/';
```

Comment:
I'm going to merge this for now and then I'll try to update the component so they accept a new site to concat to the url as a parameter so we can use it for this too